### PR TITLE
More fixes for random unit tests

### PR DESCRIPTION
**Pull request by lzap**
Should pass except the one that fails in core currently:

http://ci.theforeman.org/job/test_plugin_matrix/1393/database=postgresql,ruby=2.3,slave=fast/testReport/junit/(root)/HostJSTest__hosts%20index%20multiple%20actions/test_0001_show_action_buttons/

### CODE REVIEW

**Comment by lzap** [comment]
This hopefully fixes the last bunch @dLobatog

File: test/unit/discovered_extensions_test.rb [hunk 1]
```diff
@@ -4,6 +4,8 @@ class FindDiscoveryRulesTest < ActiveSupport::TestCase
   include Foreman::Controller::DiscoveredExtensions
 
   setup do
+    SETTINGS[:organizations_enabled] = true
```
Comment (dLobatog) [inline test/unit/discovered_extensions_test.rb]: This isn't needed in other tests, we always test with taxonomies on and don't guarantee tests will pass with taxonomies off afaik. Tests pass on the latest branch for both discovery and foreman without this, any ideas to reproduce the failure?

Comment (lzap) [inline test/unit/discovered_extensions_test.rb]: I think I found it. One test turned off organizations, looks like some tests are being scheduled after this one.

**Comment by lzap** [comment]
Looks like it, there is only this weird JS error: http://ci.theforeman.org/job/test_plugin_matrix/1395/

```
HostJSTest::hosts index multiple actions.test_0001_show action buttons
```

https://github.com/theforeman/foreman/pull/3476

**Comment by dLobatog** [comment]
Merged as fbd492f915df5b6fdd990678d2373db9ab449c79, thanks @lzap!

**Pull request closed without merging**